schema/deployment_options.schema.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
@malomarrec schema/features.schema.json should probably change too to account for the new possibilities in that file.

Reply:
ty!